xbmc/cores/VideoPlayer/VideoRenderers/OverlayRenderer.cpp

```diff
@@ -170,36 +172,46 @@ void CRenderer::Render(COverlay* o)
   COverlay::EPosition pos   = o->m_pos;
   COverlay::EAlign    align = o->m_align;
 
-  if(pos == COverlay::POSITION_RELATIVE)
+  if (pos == COverlay::POSITION_RELATIVE)
   {
     float scale_x = 1.0;
     float scale_y = 1.0;
+    float scale_w = 1.0;
+    float scale_h = 1.0;
 
-    if(align == COverlay::ALIGN_SCREEN
-    || align == COverlay::ALIGN_SUBTITLE)
+    if (align == COverlay::ALIGN_SCREEN || align == COverlay::ALIGN_SUBTITLE)
     {
-      scale_x = m_rv.Width();
-      scale_y = m_rv.Height();
+      scale_x = scale_w = m_rv.Width();
```

Comment:
I know it will look worst than it is right now but the default Kodi style around here is to keep a single assignment per line

Reply:
I've often seen this style around kodi, so I thought it would be ok, but it is not a problem change it